src/components/ContactCard/ContactForm/ContactFormField.jsx

```diff
@@ -79,7 +72,7 @@ class ContactFormField extends React.Component {
   renderArrayField = (fields, index, nameWithIndex, name, renderInput) => {
     const hasValue = fields.value[index] && fields.value[index][name]
     return (
-      <div key={nameWithIndex} className="contact-form__inputs-line">
+      <div key={nameWithIndex} className="u-mt-1 u-flex u-pos-relative">
```

Comment:
Nit: I'm not sure how applicable it is here, but we do have a [`Stack` component](https://docs.cozy.io/cozy-ui/react/#/Basics?id=stack) in cozy-ui to avoid specifying margins manually.

Reply:
Ok j'ai vu. Ca peut être bien intéressant pour la refacto de tout ça 👍